msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

```diff
@@ -146,19 +146,18 @@ There are two header variants. When possible, use the variant with the avatar to
 
 In-meeting dialogs can vary in size to account for different scenarios. Make sure to maintain padding and component sizes.
 
-* **Width**: The iframe width of the dialog is an absolute value within the range you specify.
-* **Height**: The iframe height of the dialog is an absolute value within the range you specify.
+* **Width**: You can specify the width of the dialog's iframe anywhere within the supported size range.
+* **Height**: You can specify the height of the dialog's iframe anywhere within the supported size range. Users can scroll vertically if your app content exceeds the maximum height.
```

Comment:
```suggestion
* **Height**: You can specify the height of the dialog's iframe anywhere within the supported size range. You also can allow users to scroll vertically if your app content exceeds the maximum height.
```
@KirtiPereira I changed the second sentence based on a discussion with @aosolis. Vertical scrolling is supported, but it's up to the dev to set up.